en/admin/admin_utils.rst

```diff
@@ -3343,6 +3344,137 @@ The following shows [options] available with the **cubrid vacuumdb** utility.
 
         cubrid vacuumdb -C demodb
 
+.. _flashback:
+
+flashback
+---------
+
+The **cubrid flashback** utility is used to get SQL statements to flashback a specific committed transaction, and can only be executed by the **DBA** user. The system parameter **supplemental_log** must be turned on, and flashback is supported only for DML executed after the **supplemental_log** is set. ::
```

Comment:
as @wrlawodms mentioned, probably  case ofthe word 'flashback'  using  as a verb is not that common, but it did exist. (http://www.dba-oracle.com/t_flashback_database.htm). 

However, in other official documentation (https://docs.oracle.com/cd/B19306_01/server.102/b14200/statements_9011.htm, https://docs.oracle.com/cd/E11882_01/backup.112/e10643/rcmsynta023.htm#RCMRF194), the 'flashback' as a verb is commonly replaced by words like  'rewind to '/'return to'. so I would give my suggestion to: 

 ```suggestion
The **cubrid flashback** utility is used to get SQL statements to rewind to a  specific committed transaction, and can only be executed by the **DBA** user. The system parameter **supplemental_log** must be turned on, and flashback is supported only for DML executed after the **supplemental_log** is set. ::
```

Reply:
Thanks for searching the word 'rewind'. I think it is good to use. 
Since transaction does not mean a specific time, `rewind a transaction` is better than `rewind to a transaction`. Thank you so much for finding the right word.